Makefile

```diff
@@ -1,38 +1,34 @@
+## DEVELOPMENT
+.PHONY: build
+build: 
+	@go build
 
-TAG=$(shell git describe --abbrev=0 --tags)
-
-NAME=conftest
-IMAGE=openpolicyagent/$(NAME)
-ALT_IMAGE=instrumenta/$(NAME)
+.PHONY: test
+test: 
+	@go test -v ./...
 
-COMMAND=docker
-BUILD=DOCKER_BUILDKIT=1 $(COMMAND) build --pull
```

Comment:
By removing this the build will be much slower, and not able to take advantage of the graph or separate targets. The build will always try and run every stage.

Reply:
Not quite understanding. I believe the build will always be attempted, but if the cache wasn't invalidated it'll just pick up the cached version. I'm seeing similar behaviors calling docker/buildkit directly as well as command variables.  We're not using the Make commands during CI (though we should), so I don't think there would be a huge swing in either case. The real motivation here was just easier understanding/readability.